elements/pfe-sass/variables/_colors.scss

```diff
@@ -32,7 +32,7 @@ $pfe-color--ui-link--on-dark--visited:          lighten($pfe-color--ui-link--vis
 $pfe-color--ui-link--on-dark--hover:            lighten($pfe-color--ui-link--on-dark, 10%) !default; // Was #c0ecff
 $pfe-color--ui-link--on-dark--focus:            lighten($pfe-color--ui-link--on-dark, 10%) !default; // Was #c0ecff
 
-$pfe-color--ui-link--on-saturated:              lighten($pfe-color--ui-link, 40%) !default;               // Was #97daff
+$pfe-color--ui-link--on-saturated:              $pfe-color--white;               // Was #97daff
```

Comment:
This is a system change, but it makes more sense to me if no one objects?

Reply:
👍